lib/vsc/filesystem/quota/tools.py

```diff
@@ -378,16 +379,6 @@ def process_fileset_quota(storage, gpfs, storage_name, filesystem, quota_map, cl
 
     return exceeding_filesets
 
-
-def map_uids_to_names():
```

Comment:
i would not delete this. actually, this is the sssd cache, it might not contain all users; but after a while it will.
you can check if a uid is in here first, and if not, then do the individual lookup; and only then conclude it's not an active user

Reply:
But it never contained any vsc users when testing.

Reply:
hmm, let me check that. i must be missing something then

Reply:
indeed. very weird. open me a jira for this, so i can investiagte why this is not showing up.
you can delete all this code

Reply:
We can just have the backup. Just in case.

Reply:
https://jira.ugent.be/browse/HPC-9425